### DISCUSSION TITLE
Language choice problem

### DISCUSSION THREAD

**Discussion by lhDream** [AlmasB/FXGL#1170] [Q&A] [answered]
&emsp;&emsp;I hope to use the text of the corresponding country in the interface language selection box when selecting the language of different countries.
&emsp;&emsp;For example：
```java
language: English
          Français
          简体中文
```
&emsp;&emsp;instead of
```java
language: English
          French
          chinese
```
&emsp;&emsp;What should I do?

**Comment by AlmasB** [comment, marked as answer]
Hi, are you referring to the built-in menu and its language selection? If so, there are two possible solutions:

1. you can override the existing menu with your own custom one via `SceneFactory`. See [here](https://github.com/AlmasB/FXGL/wiki/Customizing-Menus-%28FXGL-11%29), or

2. create an issue, referring to this discussion post, so it can be tracked and implemented in the future as part of the default menu.

**Comment by AlmasB** [comment]
![Screenshot 2022-07-20 21 21 00](https://user-images.githubusercontent.com/3594742/180074739-f514433e-cf83-40a1-8c5e-e2531f198ad7.png)

Selection using the target language is now supported by default.

**Reply by lhDream** [reply]
amazing  :+1:
